packages/calcite-components/src/components/table/table.tsx

```diff
@@ -92,6 +92,16 @@ export class Table extends LitElement {
    */
   @property() caption: string;
 
+  /**
+   * Sets/gets the current page
+   */
+  @property({ reflect: true }) get currentPage(): number {
+    return Math.ceil(this.pageStartRow / this.pageSize);
```

Comment:
Can you use an internal variable to store and update the current page instead of computing it in the getter (per [Lumina guidance](https://qawebgis.esri.com/components/lumina/properties#getset-properties))? Since `pageStartRow` depends on a `willUpdate` pass, getting the page immediately after setting it will produce an incorrect value. You can test this with the following:

```js
// using HTML from test
const table = document.querySelector("calcite-table");
console.log(table.currentPage, "before"); // 2
table.currentPage = 100;
console.log(table.currentPage, "after"); // 100
requestAnimationFrame(() =>
  console.log(table.currentPage, "after requestAnimationFrame"); // 3
);
```